utils/navigator_gui/navigator_gui/dashboard.py

```diff
@@ -6,11 +6,11 @@
 '''
 
 
+import functools
 import os
-import socket
-import subprocess
```

Comment:
I should point out that these were just moved to the host monitor, not removed all together.